Configuration/Geometry/python/dict2021Geometry.py

```diff
@@ -378,7 +378,6 @@
             'from Geometry.TrackerGeometryBuilder.trackerParameters_cff import *',
             'from Geometry.TrackerNumberingBuilder.trackerTopology_cfi import *',
             'from Geometry.TrackerGeometryBuilder.idealForDigiTrackerGeometry_cff import *',
-            'trackerGeometry.applyAlignment = cms.bool(False)',
```

Comment:
I am not sure I understand this change, I think it is already in master,

https://github.com/cms-sw/cmssw/blob/master/Configuration/Geometry/python/dict2021Geometry.py#L379-L382

Reply:
Indeed, I haven't change other lines apart of adding P7. I'll clean it out.

Reply:
I still don't understand why Git is showing changes that were made in previous PRs.

Reply:
The branch appears to be based on CMSSW_12_1_X_2021-08-27-1100, and the shown diff is wrt. instead of `master`. These lines have been changed both in this and `master` branch in between, so technically there could be a merge conflict. Apparently this case is straightforward-enough (the change is the same) that GitHub would be able to merge the branch into `master`.

As @perrotta pointed out, rebasing to a recent IB should remove such lines in the diff.